Tasks/Maven/pmdForMaven.ts

```diff
@@ -41,9 +41,7 @@ export function processPmdOutput(sourcesDirectory:string) : ar.AnalysisResult {
 function processPmdHtml(analysisResult:ar.AnalysisResult, path:string):ar.AnalysisResult {
     // Task fails if the file cannot be found.
     if (!tl.exist(path)) {
-        tl.error("Could not find PMD HTML output at expected location: " + path);
-        tl.error("Check that PMD ran successfully and that it is writing to the default location.");
-        tl.exit(1);
+        throw new Error("Could not find PMD HTML output. Check that PMD ran successfully and that it is writing to the default location.");
```

Comment:
Changed to throwing errors instead of calling the task library error method to minimise coupling and improve testing.